### PR TITLE
feat(sidebar): Match code color to sidebar fg, if set

### PR DESCRIPTION
**Pull request by gadenbuie**
We noticed this example in the shinycoreci tests (`308-sidebar-kitchen-sink`), and I this is a minimal fix.

I do think we should consider if we want to go down this path, and how far, because there are certainly other components or styles we might want to change as well (e.g. the link or button styles in the screenshot below)

image

### CODE REVIEW

File: inst/components/scss/sidebar.scss [hunk 1]
```diff
@@ -81,6 +81,10 @@ $bslib-sidebar-column-sidebar: Min(calc(100% - var(--_icon-size)), var(--_sideba
     background-color: var(--_sidebar-bg);
     backdrop-filter: blur(5px);
 
+    code {
+      color: var(--_sidebar-fg);
+    }
```
Comment (cpsievert) [inline inst/components/scss/sidebar.scss:86]:
```suggestion
    * {
      color: inherit;
    }
```

Comment (cpsievert) [reply inst/components/scss/sidebar.scss:86]: This won't work for hyperlinks...

**Comment by cpsievert** [comment]
I think it could maybe make sense to something like this?

```css
> .sidebar {
  --bs-body-bg: var(--_sidebar-bg);
  --bs-body-color: var(--_sidebar-fg);
}
```

Or put a `data-bs-theme="dark"` attribute on the sidebar?

**Comment by gadenbuie** [comment]
I'm going to close this in favor of using the `data-bs-theme="dark"` on the sidebar element, which I think we should promote via documentation rather than add automatically (it's a likely fix but isn't guaranteed to be correct in all cases).

https://github.com/rstudio/bslib/issues/867

**Pull request closed without merging**